6-magic.md

```diff
@@ -50,48 +48,15 @@
-
-#### Allocated spells
-
-The chart below summarises the different kinds of magic and explains who can use what spells.
-
-
-| Type of Magic | Wizard |
-| :-: | :-: |
-| Prayers of Sigmar | Sisters of Sigmar, Warrior Priests, Monks & Clerics |
-| Prayers of Shallaya | Priestess of Shallaya & Clerics |
-| Prayers of Taal | Priest of Taal & Cleric
-| Prayers of Ulric | Wolf Priest & Cleric |
-| Funerary Rites | Priest of Morr |
-| Prayers of Ranald | Priest of Ranald |
-| Norse Runes | Norse Shaman |
-| Shadow Warrior Magic | Shadow Weaver |
-| Fey Magic| Spellsinger |
-| Spells of the Djed’hi | Elf Mage |
-| Lesser Magic | Warlocks |
-| Charms and Hexes | Witches |
-| Dark Elf Magic | Dark Elf Sorceresses |
-| Necromancy | Necromancers |
-| Chaos Rituals | Chaos Magisters, Beastmen Shamans |
-| Nurgle Rituals | Carnival Masters |
-| Magic of the Horned Rat | Eshin Sorcerers, Vermin Lord |
-| Call of the Pack | Harbinger of Mutation, Vermin Lord |
-| Prayers of Corruption | Festering Chantor, Vermin Lord |
-| Machine Curses | Chief Engineer, Warlock engineer, Vermin Lord |
-| Waaagh! Magic | Orc Shamans |
-| Forest Goblin Spells | Forest Goblin Shamans |
-| Magic of the Steppes | Hobgoblin Shamans |
-| Lore of Light | Truthsayer |
-| Lore of Darkness | Dark Emissary |
+There have always been those who have used magic: witches and wizards, wise women and warlocks. But as all scholars know that all magic is dangerous, for it originates from Chaos, the source of corruption and change. Indeed, during these times, sorcery is illegal and punishable by death. Wizards have little say in whether the world of magic touches them or not. They are born with second sight and to them the world of magical energies and spells is much more real than the mundane world of normal mortals. With all the persecution, fear and hatred it is little wonder that wizards become reclusive and suspicious, and many are downright insane. Some even turn to the worship of the dark gods, and others follow the forbidden path of necromancy. Mordheim has become home to many magic users and they roam the ruins hiding from Witch Hunters. But there are other ways of gaining power over and above that of a normal mortal man. The gods watch over their subjects and a priest of strong faith can call upon them to assist them in battle. Of these the Priests of Sigmar are the most common, for Sigmar is the patron god of the Empire and his following is strong during these times of strife.
```

Comment:
```suggestion
There have always been those who have used magic: witches and wizards, wise women and warlocks. But as all scholars know that all magic is dangerous, for it originates from Chaos, the source of corruption and change. Indeed, during these times, sorcery is illegal and punishable by death. 

Wizards have little say in whether the world of magic touches them or not. They are born with second sight and to them the world of magical energies and spells is much more real than the mundane world of normal mortals. With all the persecution, fear and hatred it is little wonder that wizards become reclusive and suspicious, and many are downright insane. Some even turn to the worship of the dark gods, and others follow the forbidden path of necromancy. Mordheim has become home to many magic users and they roam the ruins hiding from Witch Hunters. But there are other ways of gaining power over and above that of a normal mortal man. The gods watch over their subjects and a priest of strong faith can call upon them to assist them in battle. Of these the Priests of Sigmar are the most common, for Sigmar is the patron god of the Empire and his following is strong during these times of strife.
```

Reply:
"Wizards have little say in whether the world of magic touches them or not." - can remove the 'or not' in that sentence now that I read it again.

Good fragmentation.